README.md

```diff
@@ -30,7 +30,6 @@ However, this has the disadvantage that you will need to commit the copied files
 ln -s ../node_modules/loam/lib/loam-worker.js public/loam-worker.js
 ln -s ../node_modules/gdal-js/gdal.wasm public/gdal.wasm
 ln -s ../node_modules/gdal-js/gdal.data public/gdal.data
-ln -s ../node_modules/gdal-js/gdal.wasm public/gdal.wasm
```

Comment:
Good catch, thanks for opening this! There does actually need to be a third line, but it obviously shouldn't be trying to make the same symbolic link over again. This line should specify `gdal.js` instead of `gdal.wasm` since that's already covered. So this should be:
`ln -s ../node_modules/gdal-js/gdal.js public/gdal.js`

If you can make that change I'll go ahead and merge.

Reply:
Done